core/store/orm/locking_strategies.go

```diff
@@ -76,14 +78,46 @@ func (s *PostgresLockingStrategy) Lock(timeout models.Duration) error {
 	}
 
 	if s.config.locking {
-		_, err := s.conn.ExecContext(ctx, "SELECT pg_advisory_lock($1)", s.config.advisoryLockID)
+		logger.Debug("Waiting for lock...")
```

Comment:
Hmm, the secondary node is going to be spamming these messages every 30s. I wonder if there's a way to log this just once when you start waiting for a lock and then log again once you got it but our logs are already crazy so 🤷‍♂️

Reply:
I feel like this is okay at DEBUG and maybe something we could clean up in a future release?

Reply:
Nope, it only prints this message once then goes into a silent loop